specification/ml/_types/TrainedModel.ts

```diff
@@ -139,3 +141,19 @@ export class TotalFeatureImportanceStatistics {
   /** The minimum importance value across all the training data for this feature. */
   min: integer
 }
+
+export enum TrainedModelType {
+  /**
+   * The model definition is an ensemble model of decision trees.
+   */
+  tree_ensemble = 0,
```

Comment:
There is no need to add the `= 0` (and so on), I'll do a pr soon to cleanup this usage pattern that we inherited from .NET.

Reply:
Done!